presto-parser/src/main/java/com/facebook/presto/sql/ReservedIdentifiers.java

```diff
@@ -40,8 +40,11 @@
     private static final Pattern IDENTIFIER = Pattern.compile("'([A-Z_]+)'");
     private static final Pattern TABLE_ROW = Pattern.compile("``([A-Z_]+)``.*");
     private static final String TABLE_PREFIX = "============================== ";
-
     private static final SqlParser PARSER = new SqlParser();
+    private static final Set<String> RESERVED_IDENTIFIERS = possibleIdentifiers().stream()
+            .filter(ReservedIdentifiers::reserved)
+            .sorted()
```

Comment:
What's the point of sorting before collecting into a set?

Reply:
Good catch, I didn't notice that the existing code that I moved here had this.  Sort is not required here.